core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -625,10 +626,19 @@ private class SchedulerPool(name: String) extends LiveEntity {
 
 }
 
-private object LiveEntityHelpers {
+private[spark] object LiveEntityHelpers {
 
   private val stringInterner = Interners.newWeakInterner[String]()
 
+  private def accuValuetoString(value: Any): String = value match {
+    case list: java.util.List[_] =>
+      if (list.size() > 5) {
```

Comment:
Does 5 looks good enough in UI? I haven't tested it by myself but thought it's faster just to ask.

Reply:
This is a number for avoiding OOM issue. For UI, good or not might be case by case. Although it sounds okay to add a config, I feel it is not a big deal to adjust this number in UI and api. I suggest to leave it as 5 and if we need to configure it, we can add a config later.